primitives/src/sequence.rs

```diff
@@ -234,3 +234,38 @@ impl fmt::Debug for Sequence {
 
 #[cfg(feature = "alloc")]
 units::impl_parse_str_from_int_infallible!(Sequence, u32, from_consensus);
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    const MAXIMUM_ENCODABLE_SECONDS: u32 = u16::MAX as u32 * 512;
+
+    #[test]
+    #[cfg(feature = "alloc")]
+    fn from_seconds_floor_success() {
+        let expected = Sequence::from_hex("0x0040ffff").unwrap();
+        let actual = Sequence::from_seconds_floor(MAXIMUM_ENCODABLE_SECONDS + 511).unwrap();
+        assert_eq!(expected, actual);
```

Comment:
```suggestion
        assert_eq!(actual, expected);
```

We try to always put these in this order so that when debugging one knows that actual is on the left without having to check for each unit test which is which. The mnemonic to help remember is its in alphabetic order (works for `got` / `want` as well).

Reply:
ah gotcha I'll keep it in that order from now on